### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by mcarr823**
New builds of Raspberry Pi OS based on Debian 12 now require slightly different packages to work with PaperTTY.

libfreetype-dev is required before the installation of pillow, otherwise it will throw errors when you try to use PIL.

libtiff5-dev is required instead of libtiff5 since the latter isn't in the repos anymore.
Installing the -dev version automatically installs the appropriate version of the base libtiff package, so this works for both Debian 11 and 12-based versions of Raspberry Pi OS.

### CODE REVIEW

**Review by joukos — APPROVED** [review]
:+1:

**Comment by joukos** [comment]
@mcarr823 - btw, since I'm very short on time for this project and feel bad about it, please let me know if you'd be interested to be a maintainer of PaperTTY. You've submitted lots of code and helped people with their issues, and I appreciate that very much.

**Comment by mcarr823** [comment]
> @mcarr823 - btw, since I'm very short on time for this project and feel bad about it, please let me know if you'd be interested to be a maintainer of PaperTTY. You've submitted lots of code and helped people with their issues, and I appreciate that very much.

Sure, I'd be happy to. I probably won't get much done over the next week or two (busy with Christmas stuff) but after that I do have some plans for further updates to PaperTTY. I might write up a TODO issue with a list...

**Comment by joukos** [comment]
> Sure, I'd be happy to. I probably won't get much done over the next week or two (busy with Christmas stuff) but after that I do have some plans for further updates to PaperTTY. I might write up a TODO issue with a list...

:pray: That's great, thanks! I invited you as collaborator now (I guess that should work? It's the first time I do this in Github). The only expectation/hope I have is that you don't feel at all stressed/obliged about this - it's just more convenient if someone who actually works on the code doesn't have to bother waiting for me in particular to merge good PRs etc. :)

Let me know if there's any problems pushing to the repo or such.

**Comment by mcarr823** [comment]
Thanks @joukos invite accepted.
I won't stress about it. I do have a bunch of additions planned for next year, but I'll just do it when I have the time.

I'll still probably submit a PR if it's something other than straightforward code additions.
eg. I think the README, installation methods and dependencies are due for an update, and I'll likely submit a PR to get your impressions on that.
But for normal code changes I'll merge them in myself.

**Pull request merged**
